Animations/Animation.js

```diff
@@ -0,0 +1,126 @@
+// 'use strict';
+var React = require("react/addons");
+var Arrival = require('Arrival');
+var Easings = require("./AnimationEasings.js");
+var ReactTransitionGroup = React.addons.TransitionGroup;
+
+//React.createClass FUNCTION: function to create animation component class
+  //Required keys:
```

Comment:
Indicate that the createClass function takes an object input with the required keys that you've listed.